src/scala/io/bazel/rules_scala/scrooge_support/BUILD

```diff
@@ -8,7 +8,9 @@ scala_library(
     visibility = ["//visibility:public"],
     deps = [
         ":focused_zip_importer",
+        "//external:io_bazel_rules_scala/dependency/scala/guava",
```

Comment:
is this addition and the line below needed? I don't see any source changes correlating to it

Reply:
Scrooge needs Guava and mustache to generate java code from thrift.

Reply:
But where is it used exactly?

Reply:
I honestly don't see any explicit guava imports, but the `ApacheJavaGenerator` class mentioned in the comment about mustache fails without this import.

Reply:
Oh, ApacheJavaGenerator apparently relies on `com/google/common/base/Function` for some reason. This line fails: https://github.com/twitter/scrooge/blob/15d061372d11cef341bc00eabd9efdfa79920cf0/scrooge-generator/src/main/scala/com/twitter/scrooge/java_generator/ApacheJavaGenerator.scala#L297

With the following error:
```
Exception parsing: test/src/main/scala/scalarules/test/twitter_scrooge/thrift/thrift2/thrift3/Thrift3.thrift
com.twitter.scrooge.frontend.FileParseException: Exception parsing: test/src/main/scala/scalarules/test/twitter_scrooge/thrift/thrift2/thrift3/Thrift3.thrift
Caused by: java.lang.NoClassDefFoundError: com/google/common/base/Function
        at com.twitter.scrooge.java_generator.ApacheJavaGenerator.$anonfun$apply$2(ApacheJavaGenerator.scala:297)
        at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
        at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
        at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
        at com.twitter.scrooge.java_generator.ApacheJavaGenerator.apply(ApacheJavaGenerator.scala:294)
        at io.bazel.rules_scala.scrooge_support.Compiler.$anonfun$run$6(Compiler.scala:149)
        at io.bazel.rules_scala.scrooge_support.Compiler.$anonfun$run$6$adapted(Compiler.scala:120)
        at scala.collection.Iterator.foreach(Iterator.scala:943)
        at scala.collection.Iterator.foreach$(Iterator.scala:943)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
        at io.bazel.rules_scala.scrooge_support.Compiler.run(Compiler.scala:120)
        at scripts.ScroogeWorker$.work(ScroogeWorker.scala:93)
        at io.bazel.rulesscala.worker.Worker.persistentWorkerMain(Worker.java:92)
        at io.bazel.rulesscala.worker.Worker.workerMain(Worker.java:46)
        at scripts.ScroogeWorker$.main(ScroogeWorker.scala:15)
        at scripts.ScroogeWorker.main(ScroogeWorker.scala)
Caused by: java.lang.ClassNotFoundException: com.google.common.base.Function
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
```